src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/create-cluster/create-cluster.component.ts

```diff
@@ -140,12 +157,37 @@ export class CreateClusterComponent implements OnDestroy {
           }
         });
     }
+
+    if (this.deploymentOption) {
+      const trackingId = `${this.deploymentOption} optimized deployment`;
```

Comment:
Is this text that's going to be presented to the user? If so, it should be localized.

Reply:
Done!